### PR TITLE
Jetpack Section: add tracking for Jetpack Scan events

### PR DESCRIPTION
**Pull request by momo-ozawa**
Part of #15190

### Description:
- Added tracking for Jetpack Scan events 👀 

### To test:

Verify that the following actions fire the correct event with the correct properties.

Please refer to the event tracking spreadsheet for more details about the event name / event properties.

Action | Event
--- | ---
Open the scan screen | `jetpack_scan_accessed`
Tap the history button | `jetpack_scan_history_accessed`
Switch tabs on the scan history screen | `jetpack_scan_history_filter`
Go back to scan screen and tap on scan now | `jetpack_scan_run_tapped`
Tap on a threat | `jetpack_scan_threat_list_item_tapped`
Tap on ignore threat | `jetpack_scan_ignorethreat_dialogopen`
Confirm ignore on alert dialog | `jetpack_scan_threat_ignore_tapped`
Tap on a threat then tap on fix threat | `jetpack_scan_fixthreat_dialogopen`
Confirm fix on alert dialog | `jetpack_scan_threat_fix_tapped`
Tap on fix all in scan screen | `jetpack_scan_allthreats_open`
Tap on contact support in scan screen (scan error) | `jetpack_scan_error_contact_tapped`

#### Errors
Error | Event
--- | ---
Scan error (turn wifi off mid process) | `jetpack_scan_error`
Ignore error (turn wifi off mid process) |  `jetpack_scan_error`
Fetch scan history error (turn wifi off mid process) | `jetpack_scan_error`

### PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by momo-ozawa** [comment]
@emilylaguna 👋 
I left a couple of Scan events to be added later, since I wanted to wait until your current PRs are merged.

**Review by emilylaguna — APPROVED** [review]
LGTM. I'll add the missing events in a new PR.

**Pull request merged**
